Node/Managing Node-NPM versions.md

```diff
@@ -0,0 +1,127 @@
+## Node version management
+
+### Node release schedule
+
+Node.js has defined and predictable release plan. Each node version goes through 3 phases: current, active(LTS) and maintenance. Be aware that odd-numbered versions will not go through active and maintenance phases.
+
+#### Current phase
+
+Incorporates most non-major changes that end up on `nodejs/node` main branch. New major releases are branched out every six months. Even-numbered versions are scheduled for release in April, while odd ones are scheduled for October.
+
+#### Active Long Term Support phase
+
+Versions in this phase include new features, bug fixes and updates that have been audited and determined to be stable for the release. With each odd-numbered release, previous even-numbered release will transition to LTS phase. Versions in this phase will receive active support for 12 months after they entered LTS phase.
+
+**You should always aim to use latest LTS version available on your project**
+
+#### Maintenance phase
+
+Versions in this phase will receive critical bug fixes and security updates for 18 months after it enters this phase.
```

Comment:
```suggestion
Releases within this phase will receive critical bug fixes and security updates for 18 months after it enters this phase.
```

Reply:
Actually, it's 18 moths for LTS maintenance, but only 2 months for regular maintenance (odd versions).